DuckDuckGo/HomePageConfiguration.swift

```diff
@@ -45,17 +56,18 @@ class HomePageConfiguration {
     }
     
     enum Component: Equatable {
-        case navigationBarSearch
+        case privacyProtection
+        case navigationBarSearch(withOffset: Bool)
         case centeredSearch(fixed: Bool)
-        case favorites
-        case padding
+        case favorites(withHeader: Bool)
+        case padding(withOffset: Bool)
         case empty
```

Comment:
These `withOffset`/`withOffset` parameters should disappear once experiment is done.